### PR TITLE
[move][cli] Remove Movey and sandbox from CLI

### PR DESCRIPTION
**Pull request by georgemitenkov**
## Description

Move CLI has:
- Movey, previously used by Sui and is now shut down (check out https://www.eastagile.com/blogs/movey-net-status)
- Sandbox, to run Move commands locally.

This PR removes both and moves a few testing utilities (defined in `sandbox` crate for unknown reason) into a new `test` file. Th rationale is:
- There are no code owners of this code
- This is legacy code
- This code is not used (Movey, 100%, instead of sanbox there are transactional tests and Aptos CLI)
- This code hardcodes VM types which makes any changes to VM and new integration harder. Mostly people hack to make things work but no one knows if that breaks sandbox, etc.

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [x] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: third_party/move/tools/move-cli/tests/sandbox_tests/module_disassemble/Move.toml
Comment (wrwg) [inline third_party/move/tools/move-cli/tests/sandbox_tests/module_disassemble/Move.toml:1]: Lets open a bug to get disassembler test back somewhere else

File: third_party/move/tools/move-cli/tests/sandbox_tests/print_stack_trace/args.v2_exp
Comment (wrwg) [inline third_party/move/tools/move-cli/tests/sandbox_tests/print_stack_trace/args.v2_exp:1]: Need a bug to bring this test back

**Review by wrwg — APPROVED** [review]
I glanced over tests, and it appears most are covered (made a comment where not):

- publishing is tested in e2e-move-tests and anyway this publishing here is outdated
- aborts et. al are tested in various e2e tests
- gas is tested elsewhere in e2e tests

So I think we can go ahead with this.

**Review by vgao1996 — APPROVED** [review]

File: third_party/move/tools/move-cli/tests/sandbox_tests/module_disassemble/Move.toml
Comment (georgemitenkov) [reply third_party/move/tools/move-cli/tests/sandbox_tests/module_disassemble/Move.toml:1]: Done

File: third_party/move/tools/move-cli/tests/sandbox_tests/print_stack_trace/args.v2_exp
Comment (georgemitenkov) [reply third_party/move/tools/move-cli/tests/sandbox_tests/print_stack_trace/args.v2_exp:1]: Done

**Comment by georgemitenkov** [comment]
Thanks @wrwg @vgao1996: issues created: https://github.com/aptos-labs/aptos-core/issues/13421

**Pull request merged**